opencensus-shim/src/test/java/io/opentelemetry/opencensusshim/TraceInteroperabilityTest.java

```diff
@@ -66,8 +66,20 @@
   static {
     spanExporter = spy(SpanExporter.class);
     when(spanExporter.export(anyList())).thenReturn(CompletableResultCode.ofSuccess());
+
     SpanProcessor spanProcessor = SimpleSpanProcessor.create(spanExporter);
-    OpenTelemetrySdk.getGlobalTracerManagement().addSpanProcessor(spanProcessor);
+    openTelemetry =
+        OpenTelemetrySdk.builder()
+            .setTracerProvider(SdkTracerProvider.builder().addSpanProcessor(spanProcessor).build())
+            .build();
+    GlobalOpenTelemetry.set(openTelemetry);
+  }
+
+  @Captor private ArgumentCaptor<Collection<SpanData>> spanDataCaptor;
+
+  @BeforeEach
+  void resetMocks() {
+    reset(spanExporter);
```

Comment:
this might be a very rare case where I would use the full `Mockito.reset` since I had no idea where this method was coming from without looking at imports.

Reply:
Commenting on the merge commit :)

Not going to send a PR to rename it but if it feels any better, hopefully the method name `resetMocks` helps.